productsapp/components/Login.js

```diff
@@ -1,54 +1,98 @@
 import * as React from 'react';
-import { Text, View, TextInput, Image, TouchableOpacity } from 'react-native';
-import ProductsList from '/ProductsList';
+import {
+  Text,
+  View,
+  TextInput,
+  ActivityIndicator,
+  Image,
+  FlatList,
+  TouchableOpacity,
+  Alert,
+} from 'react-native';
 import { Font } from 'expo';
 import styles from './Styles';
-
+import ProductsList from './ProductsList';
 export default class Login extends React.Component<{}> {
-   componentDidMount() {
+  constructor(props) {
```

Comment:
you can write easily like this
```
state = {
  isLoading: true,
  username: '',
  password: '',
  fontLoaded: false
};
```
instead of constructor function